tests/test_fireforward_outputs.py

```diff
@@ -0,0 +1,180 @@
+import pytest
+import fireatlas
+import geopandas as gpd
+import os
+
+
+@pytest.fixture
+def v3_run(tmp_settings_context_manager,
+    test_data_dir):
+    # run v3 for Creek fire and store outputs locally
+    
+    with tmp_settings_context_manager(
+        fireatlas.settings,
+        LOCAL_PATH=test_data_dir, 
+        EPSG_CODE=9311, 
+        FIRE_SOURCE="SNPP", 
+        FIRE_NRT=False,
+        remove_static_sources=False, 
+        FTYP_OPT = "CA", 
+        CONT_OPT="CA"
+    ):
+
+        from fireatlas import FireMain, postprocess, FireTime, preprocess, settings
+        from fireatlas.FireLog import logger
+        import logging
+
+        logger.setLevel(logging.WARNING) 
+    
+        tst = [2020, 9, 5, "AM"]
+        ted = [2020, 11, 5, "PM"]
+        
+        region = ("v3_test_data_for_Creek_SNPP", [-119.5, 36.8, -118.9, 37.7])
+
+        # Expects that we have done preprocessing of monthly data ahead of time 
+        # and saved in test/data/FEDSpreprocessed. 
+        # To reproduce, use FireIO.convert_v2_pkl_to_csv on data saved 
+        # in FEDStemp by v2. 
+        
+        list_of_ts = list(FireTime.t_generator(tst, ted))
+
+        preprocess.preprocess_region(region, force=True) 
+
+        for t in list_of_ts:
+            preprocess.preprocess_region_t(
+                t, region=region, read_location="local", force=True
```

Comment:
There is also a global setting called `settings.READ_LOCATION` that would ensure that everything is read from local without you needing to specify in each funciont.

Reply:
Thank you! It turns out that this connects to the other problem we are having. If I set `READ_LOCATION` to local globally, the test runner will also look for landcover data locally. The preprocessed `nlcd_export_510m_simplified_latlon.tif` file is 65MB, so I was hoping to avoid committing it to the repo directly. However, if I don't, we need to give the Actions runner permissions to read it from a bucket. This question is what led to [this issue](https://github.com/Earth-Information-System/fireatlas/issues/74).

Reply:
hmm yeah that's a good point. I guess we could maybe use git-lfs but that's not a good solution

Reply:
another option would be to run it with FTYPE set to the one that doesn't use the landcover file.